app/compose/Aggregator.scala

```diff
@@ -49,12 +60,17 @@ class Aggregator @Inject()(movieSearchService: MovieSearchService,
 
   private def handleFailure(exception: Throwable): String = {
     exception match {
-      case failure: InvalidQueryParamException => s"Failure calling downstream service due to invalid query parameters supplied: ${failureInfo(failure)}"
-      case failure: GenericFailureCalling => s"Failure calling downstream service: ${failureInfo(failure)}"
-      case failure: FailureDecodingJson => s"Failure decoding json: ${failureInfo(failure)}"
-      case _@failure => s"Generic failure calling downstream service: ${failureInfo(failure)}"
+      case failure: InvalidQueryParamException =>
+        s"Failure calling downstream service due to invalid query parameters supplied: ${failureInfo(failure)}"
```

Comment:
this is not a good practice; to limit your code 80 characters. every Developer has a big screen not just the laptop; I prefer the old style everything in one line

Reply:
reformatted using scalafmt class